orchestrate-cockroachdb-with-kubernetes.md

```diff
@@ -0,0 +1,341 @@
+---
+title: Orchestrate CockroachDB with Kubernetes
+summary: 
+toc: false
+---
+
+[Kubernetes](http://kubernetes.io/) is an open-source system for automating the deployment, scaling, and management of containerized applications. Although Kubernetes is not yet production-ready for stateful applications such as CockroachDB, you can use Kubernetes' [`minikube`](https://github.com/kubernetes/minikube) tool to test out CockroachDB on Kubernetes locally, using persistent volumes and the alpha [Pet Set](http://kubernetes.io/docs/user-guide/petset/) feature to ensure persistent identity and storage for CockroachDB nodes. This page shows you how.
+
+<div id="toc"></div>
+
+## Step 1. Install and start Kubernetes locally
+
+Follow Kubernetes' [documentation](http://kubernetes.io/docs/getting-started-guides/minikube/) to install `minikube` and `kubectl` for your OS. Then start a local Kubernetes cluster:
+
+~~~ shell
+$ minikube start
+Starting local Kubernetes cluster...
+Kubectl is now configured to use the cluster.
+~~~
+
+## Step 2. Create persistent volumes and volume claims
+
+Each CockroachDB node will run in a [pod](http://kubernetes.io/docs/user-guide/pods/), which is a group of one or more Docker containers that can share resources. In our case, each pod will have only one container running a CockroachDB node. 
```

Comment:
I'm not sure what the right balance between useful detail and overly distracting from the actual example is, but it likely depends on the target demographic. Maybe someone who doesn't know kubernetes well can weigh in.